BDArmory/Radar/RadarUtils.cs

```diff
@@ -716,7 +717,54 @@ public static TargetInfo RenderVesselRadarSnapshot(Vessel v, Transform t, Target
                     v.SetPosition(v.transform.position - presentationPosition);
                 }
             }
-
+            if (!BDArmorySettings.DEBUG_RADAR)
```

Comment:
Is it intentional that this part gets called when you're _not_ debugging radar?
I suspect it is, but thought it best to ask.